### PR TITLE
Remove muliple card dashes by adding affordance for keyed configuration to configs.

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: client/chromez-api.js [hunk 1]
```diff
@@ -24,3 +24,31 @@ function removeCard(card) {
   if (cardview)
     return Polymer.Base.async(function() { cardview.removeCard(card); });
 }
+
+function addDash(dashSpec) {
+  if (typeof dashSpec == "object")
+  {
```
Comment (dstoc) [inline client/chromez-api.js]: we should set up lint

File: client/chromez-api.js [hunk 2]
```diff
@@ -24,3 +24,31 @@ function removeCard(card) {
...
+  }
+
+  var e = dashpoint.querySelector(dash);
+  if (e && e.key == key)
+    return;
+  Polymer.Base.importHref(dash + '.html',
+    function(e) {
+      var elt = document.createElement(dash);
+      elt.key = key;
+      dashpoint.insertBefore(elt, feed);
```
Comment (dstoc) [inline client/chromez-api.js]: we should probably require a container to add the dashes to, rather than require that there's an element with id=feed

**Comment by dstoc** [comment]
lgtm

**Comment by shans** [comment]
Rebased over Alan's patch and comments addressed.

**Pull request merged**